### PR TITLE
The two .deb files are now in the github releases tag-(Solves issue- #18)

### LINKED ISSUE #18: Apart from pushing to the deb branch use the Github releases deployment
At present the deb files are released to the deb branch. We should also use the Github releases tab.

### PR DESCRIPTION
**Pull request by pranav1698**
Fixes #18 

- Short description of what this resolves:
The two files generated are now released as a github release 

- Changes proposed in this pull request:
    - travis.yml is updated with a deploy section which deploys two files

You can check the releases for my local fork:
https://github.com/pranav1698/meilix-artwork/releases
https://travis-ci.org/pranav1698/meilix-artwork/builds

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
-  - bash ./scripts/deploy.sh
-
-notifications:
+- bash ./scripts/deploy.sh
+notifications: 
 slack: fossasia:3pMkQrSogvEMfnu9aJDW2ZYC
+deploy:
+  provider: releases
+  api_key:
+    secure: WdeGjOS9b7wz0jCT6OSKQNmA1gDduUbPST4+JidxTc8IsJfmfbWyuX465Phh/TEs9usE2wyvRESC1/ClD7Wdys4U1HLCNzRaKqxShFwwWYDCiHuEfKW9uAQ94xYP+4TrJHICMmvx0pFK1ZC02RXs0plw4SkIxexl6vQ1jyztkeRdFDAPvbpnla+xWlg5W5Q9PGzoareQcMVP7FM3enJuAFRMpZO/tL/0H5EPk7/E6jUf4mDvyVt2MGZZa+M0viHsQYaK+t1tssQ5XbRAKccwBd7TkkJemuce7LSouhkfbKXSri7doCFfoaLUNXlv9XL9A0MY0IWNLAWifvT7jcSQa0dsNJPbTKe8AwUnB3pBq6lDFRYZJ9IBNThMvXlpf6fjiRTfrFDDGErH2J29x1ghzIsmqB3oIiI76wwv3QTABHP+y7aMOYrhH2uPB+iQRMnA6t2JF2FzLa6BJ2YaCT/alGRU2ObmDgxk9xyypkIyvztsBHzrz5WLN2rCrB9/MOT2E/gRfNYZU2jNybmuoJOFQy3Z4sirg+9oKpK1pB1IewDRnsCbNtMOtsuTDOtxDxBJb1sPLHCUKv5uETqFYp9si6Qnxi9S2Q6SchQ++PmoWmADEMnC8nmp/+Nxgr/8APd/Vjv0Yupao//oHXmNuOo5EIeO+0oo042qgG4Mhii45Ec=
```
Comment (meets2tarun) [inline .travis.yml]: I've replaced the key with mine one

```suggestion
    secure: "Z/FXwDBbeylKP3qY8slHtM9rUzxUtGLNz0QMGefbkSrOvRseqwCFTmMiek+jBNETuEwEU057wioYULQAV/7gFX358i1VzlPMUSGSZSKSb1uPqzeUGa/I3wgOFFDcrK2ntNByWQLRlQbxubauwTo/YxIvQQI9wCC5ML41R/lw+ixuf45goPdMmbcX6TiTcNQWcbT4vC/qP5pTVVTD89gAqcmUq/gXY4OQ49Bzctna4P7AEBpk6e7FBHQG6CLgSeLTj7m/ZaRjP5Jc/PRjGGeQGhPtvXhES+gL85Dwep091iG39B60FvXDTjkhi4ObHeOlKbjf3WzaD143kl3sYhNEMfArdz3u1mAnfcVw5r11gm02sWuWqNdtAPAOFmHk/bqtOE8Va7c0WlhvDjrlrqBiCmy3Prr756bZ7x0HjuivBtFB7NudcugeQVHBls9G9NfQkf9quXtTlLdfjABvgaRUZ843rv6uLuvbmRAJ6lOvXIYF9Z5TwoKZ59Z/BAD4QClcWQRXDclD5cnCzZSFS/aGdqya77g7NS0oVFKxjjyrdU4vQh1WrgtGL3dOAi49e+NDqvztUk8UJ/+8Zw7QdydKGyA2pAIFv5kx/8i6dF1EF2XMJ2CVnvuvloO/MEjm3h68jL7op+zZ1oEjCaHZ2746k0G3ZCC7e54uWICZwoWkhWc="
```

File: .travis.yml [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
+  file:
+    - "./plymouth-meilix-logo_1.0-1_all.deb"
+    - "./plymouth-meilix-text_1.0-1_all.deb"
+  on:
+    repo: pranav1698/meilix-artwork
```
Comment (meets2tarun) [inline .travis.yml]:
```suggestion
    repo: fossasia/meilix-artwork
```

File: .travis.yml [hunk 3]
```diff
@@ -1,22 +1,28 @@
...
+    branch: deploy-debfiles
```
Comment (meets2tarun) [inline .travis.yml]:
```suggestion
    branch: master
```

**Comment by pranav1698** [comment]
What are the changes I need to make @meets2tarun?

File: .travis.yml (hunk 3 above)
Comment (tabesin) [reply .travis.yml]: we currently have a deb branch. generated filed should not go to master. In the worst case the build process may loop.

**Comment by pranav1698** [comment]
@tabesin @meets2tarun @abishekvashok please review now

File: meilix-artwork_1.0-1_amd64.build [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+dpkg-source: info: building meilix-artwork in meilix-artwork_1.0-1.tar.xz
+dpkg-source: info: building meilix-artwork in meilix-artwork_1.0-1.dsc
+ debian/rules build
+dh build 
+dh: Compatibility levels before 9 are deprecated (level 8 in use)
+   dh_update_autotools_config
+   dh_auto_configure
+dh_auto_configure: Compatibility levels before 9 are deprecated (level 8 in use)
+   debian/rules override_dh_auto_build
+make[1]: Entering directory '/home/pranav1698/Desktop/open-source/meilix/meilix-artwork/meilix-artwork'
```
Comment (meets2tarun) [inline meilix-artwork_1.0-1_amd64.build]: This file contains lots of address of your personal system.

I think this file will get deleted and new one will generate, so why don't we use debuild script for the same.

Did you get my point? @pranav1698

Comment (pranav1698) [reply meilix-artwork_1.0-1_amd64.build]: Yeah, I think this file gets generated every time we run debuild.sh, maybe deleting the generated files just for merging will be make the task clear

**Comment by tabesin** [comment]
Add the files generated locally to .gitignore then. If the issue here are just files thst get overwritten anyway then Approve PR (and maybr delete these files from the repo), in any case add .gitignore -- If that needs a ticket create it please. 

Otherwise delete the fragments, add fragment deletion to debuild script and have this PF rectified.

**Comment by tabesin** [comment]
https://github.com/fossasia/meilix-artwork/issues/26 For .gitignore I opened a new ticket.

File: meilix-artwork_1.0-1_amd64.build (hunk 4 above)
Comment (tabesin) [reply meilix-artwork_1.0-1_amd64.build]: Maybe add later to .gitignore and remove the file.

**Pull request merged**